youtube/generic.md

```diff
@@ -1,37 +1,46 @@
-...
+## Youtube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://www.youtube.com/eddiejaoude?sub_confirmation=1
 
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
+**Make sure to tell your friends about it, so that they also learn something new!**
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+## Youtube Shorts
+I have another youtube channel for short clips, please **take a moment subscribing it: https://www.youtube.com/eddiejaoudetv**
 
-Alslo join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at https://www.eddiehub.org
+
+Make sure to checkout my website and sign up to our community **newsletter**: https://www.eddiejaoude.io for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organisation**:
+
+- Go to https://www.github.com/EddieHubCommunity/support/issues/new/choose 
+- Choose The Issue Type Of **Invite Me To The Organisation**
+- Open The Issue
+- You get an invitation to the community by our GitHub Action 🎉
 
 ---------------------------------------------------------------------------------------------------
 
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
+## Big Thanks To Sponsors
```

Comment:
Perhaps we can mention `Github Sponsors`, in order to convey the platform of sponsorship?